index.js

```diff
@@ -1617,6 +1635,10 @@ GenericPLCAccessory.prototype = {
     return 100-value;
   },
 
+  invert_bit: function(value) {
+    return 1 - value;

```

Comment:
As in theory true is anything which is not zero I would prefer this.
```suggestion
    return (value ? 0 : 1);
```